x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/WaitForNoFollowersStep.java

```diff
@@ -49,10 +53,31 @@ public boolean isRetryable() {
 
     @Override
     public void evaluateCondition(Metadata metadata, Index index, Listener listener, TimeValue masterTimeout) {
+        XPackInfoRequest xPackInfoRequest = new XPackInfoRequest();
+        xPackInfoRequest.setCategories(EnumSet.of(XPackInfoRequest.Category.FEATURES));
+        getClient().execute(
+            XPackInfoAction.INSTANCE,
+            xPackInfoRequest,
+            ActionListener.wrap((xPackInfoResponse) -> {
+                XPackInfoResponse.FeatureSetsInfo featureSetsInfo = xPackInfoResponse.getFeatureSetsInfo();
+                if (featureSetsInfo != null) {
+                    XPackInfoResponse.FeatureSetsInfo.FeatureSet featureSet = featureSetsInfo.getFeatureSets().get(CCR_LEASE_KEY);
+                    if (featureSet != null && (featureSet.available() == false || featureSet.enabled() == false)){
```

Comment:
Let's please just check if the featureSet is enabled here.
e.g. 
```
if (featureSet != null && featureSet.enabled() == false)
```

We'd like to maintain the same behaviour for this step even if CCR is unavailable (e.g. license is expired)

Reply:
> We'd like to maintain the same behaviour for this step even if CCR is unavailable (e.g. license is expired)

LGTM